okio/src/jvmMain/kotlin/okio/AsyncTimeout.kt

```diff
@@ -97,6 +97,28 @@ open class AsyncTimeout : Timeout() {
    */
   internal fun remainingNanos(now: Long) = timeoutAt - now
 
+  /** If scheduled, this is the expiration time of this timeout  */
+  internal fun timeoutAt(): Long = timeoutAt
```

Comment:
nit: I think callers should just read the field. It’s also internal?

Reply:
Updated!